osf/migrations/0156_fileversion_name.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2019-02-26 15:42
+from __future__ import unicode_literals
+
+import logging
+
+from django.db import migrations, models
+from django_bulk_update.helper import bulk_update
+
+logger = logging.getLogger(__file__)
+
+
+def populate_fileversion_name(state, *args, **kwargs):
+    FileVersion = state.get_model('osf.fileversion')
+    file_versions_to_update = []
+    for fv in FileVersion.includable_objects.all():
+        fv.name = fv.basefilenode_set.first().name
+        file_versions_to_update.append(fv)
+
+    bulk_update(file_versions_to_update, update_fields=['name'], batch_size=1000)
+    logger.info('Populated `name` on a total of {} file versions'.format(len(file_versions_to_update)))
```

Comment:
Populating the FileVersion's name with the file name.  

Is this an ok assumption, or do we need to go through the file target's logs to try to pull the file name? (In event of rename, for example?)

Also, this migration would be way too slow. Optimize.